tests/extras/test_websocket.py

```diff
@@ -0,0 +1,196 @@
+import socket
+import threading
+import websocket
+from pytest import mark, fixture, raises
+from urllib.parse import urljoin
+
+from yellowbox.extras.websocket import WebsocketService
+
+TEST_PATH = "/test"
+
+
+@fixture(scope="module")
+def global_websocket_service():
```

Comment:
I'd underline this

Reply:
Also why don't you make the service per-test?

Reply:
For the same reason i can ask "why would I?"